src/ui/RequestContainer.vue

```diff
@@ -84,7 +85,7 @@ export default {
   },
   data() {
     return {
-      numOfVotes: [],
+      numOfVotes: {},
```

Comment:
I think the problem was that the vote counts were getting mixed up in the array, so hopefully switching to an object fixes the bug.

Reply:
Haha, makes sense now that you point it out. Nice catch.